addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -519,28 +519,25 @@
                                 <field name="active" invisible="1"/>
                                 <field name="type" invisible="1"/>
                                 <field name="detailed_type" invisible="1"/>
-                                <field name="standard_price" help="When the cost of an expense product is different than 0, then the user using this product won't be able to change the amount of the expense, only the quantity. Use a cost different than 0 for expense categories funded by the company at fixed cost like allowances for mileage, per diem, accommodation or meal."/>
-                                <field name="uom_id" groups="uom.group_uom" options="{'no_create': True}"/>
+                                <field name="standard_price" style="width:40%%" help="When the cost of an expense product is different than 0, then the user using this product won't be able to change the amount of the expense, only the quantity. Use a cost different than 0 for expense categories funded by the company at fixed cost like allowances for mileage, per diem, accommodation or meal."/>
```

Comment:
Don't use style ==> 
"Below are the order of specificity rule which has precedence respectively: 1) Inline style: Inline style has highest priority among all. 2) Id Selector: It has second highest priority. 3) Classes, pseudo-classes and attributes: These selectors has lowest priority."

Always try to use bootstrap in a class or do a custom css class :smile: